File: Python/uberBlack.py
from car import Car
typeCarAccepted = []
seatsMaterials = []
class UberBlack(Car):
  def __init__(self, licence, driver, typeCarAccepted, seatsMaterials):
      super().__init__(licence, driver)
      self.typeCarAccepted = typeCarAccepted
      self.seatsMaterials = seatsMaterials

File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {
  Map<String, Map<String,Integer>> typeCarAccepted;
  ArrayList<String> seatsMaterial;
  public UberBlack(String licence, Drver driver, Map<String, Map<String,Integer>> typeCarAccepted,ArrayList<String> seatsMaterial){
     super(licence, driver);
     this.typeCarAccepted = typeCarAccepted;
     this.seatsMaterial = seatsMaterial;
   }
}
